README.md

```diff
@@ -40,10 +40,10 @@ If you want to upload your own modified version of the zkVM guest, you'll need t
 export PINATA_JWT="YOUR_PINATA_JWT"
 ```
 
-Then run without the `--image-url` flag:
+Then run without the `--program-url` flag:
 
 ```bash
-RUST_LOG=info cargo run --bin app -- --even-number-address ${EVEN_NUMBER_ADDRESS:?} --number 4
+RUST_LOG=info cargo run --bin app -- --rpc-url ${RPC_URL:?} --private-key ${PRIVATE_KEY:?} --even-number-address ${EVEN_NUMBER_ADDRESS:?} --number 4
```

Comment:
If `RPC_URL` is already set as an environment variable, it should be required to also specify it with `--rpc-url`. The private key and event number address work the same way (due to the `#[clap(env)]` attribute on the args.